extensions/cornerstone/src/toolbarComponents/VTKMPRToolbarComponent.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { ToolbarButton } from '@ohif/ui';
+
+let isVisible = true;
+
+const _isDisplaySetReconstructable = (studies, viewportSpecificData = {}, activeViewportIndex) => {
```

Comment:
Where did the logic for this use to live?

Reply:
This used to live into `PluginSwitch.js`